tsdate/core.py

```diff
@@ -223,9 +227,7 @@ def get_modified_ts(self, result, eps):
         meta_timing -= time.time()
         logger.info(f"Inserted node and mutation metadata in {abs(meta_timing)} seconds")
         sort_timing = time.time()
-        tables.sort()
-        tables.build_index()
-        tables.compute_mutation_parents()
```

Comment:
also we want this tables.compute_mutation_parents() I think (along with the initialization of mutation.parent to NULL above) or we can get tree sequences that error out when stats functions are called